docs/pages/machine-id/getting-started.mdx

```diff
@@ -51,8 +51,9 @@ downloads page ships with several useful binaries, including `teleport`,
 - `tbot` is the Machine ID tool you will use to associate a bot user with a machine (step 3/4)
 
 Machine ID is available starting from the Teleport `9.0.0` release. Download
-the appropriate Teleport package for your platform from our
-[downloads page](https://goteleport.com/teleport/download).
+the appropriate Teleport package for your platform.
+
+(!docs/pages/includes/install-linux.mdx!)
```

Comment:
Not sure if this one belongs here. We already list "an existing Teleport cluster" as a prerequisite, and link to the getting started guide above.

The rest of this guide is largely focused on the `tbot` binary, and the install instructions linked here talk about how to install `teleport` in general.

Further, I'm not sure the `tbot` binary is available in the DEB or RPM packages, so this might further confuse people.